django/core/handlers/wsgi.py

```diff
@@ -159,7 +159,7 @@ def __call__(self, environ, start_response):
                     # Check that middleware is still uninitialized.
                     if self._request_middleware is None:
                         self.load_middleware()
-                except:
+                except TypeError:
```

Comment:
I'm sorry. I don't know why I put there `TypeError`. Maybe, there is no need of this try block at all, because `self._request_middleware` is not `None` only when `load_middleware` finishes without exceptions.